packages/cli/src/extensions.ts

```diff
@@ -7,8 +7,12 @@ export async function configureExtensionsFromSpec(specFilePathOrObject: string |
   const result = decycle(await dereference(specFilePathOrObject));
 
   forOwn(get(result, 'x-json-schema-faker', {}), (value: any, option: string) => {
-    if (option === 'locale') return jsf.locate('faker').setLocale(value);
+    if (option === 'locale') {
+      // @ts-ignore
```

Comment:
Is it possible to get rid of this ts-ignore by defining types for the `json-schema-faker` locally? If not would be great to have a comment with an explanation why do we need it here.